pkg/scheduler/framework/plugins/noderesources/most_allocated.go

```diff
@@ -63,3 +69,32 @@ func (ma *MostAllocated) ScoreExtensions() framework.ScoreExtensions {
 func NewMostAllocated(_ *runtime.Unknown, h framework.FrameworkHandle) (framework.Plugin, error) {
 	return &MostAllocated{handle: h}, nil
```

Comment:
This will become something like this:
```
return &MostAllocated{
  handle: h,
  resourceAllocationScorer: {MostAllocatedName, mostResourceScorer, DefaultRequestedRatioResources},
 }, nil
```